bigcommerce/big-design
# CPX-160 add logout icon

## What?

Add the logout icon to big-design

## Why?

Adding an `logout` icon.

## Screenshots/Screen Recordings

<img width="226" alt="Screenshot 2023-07-05 at 3 24 18 PM" src="https://github.com/bigcommerce/big-design/assets/106188209/7679a2f0-7a54-4aec-a132-dd50cea0297c">

## Testing/Proof

- pull PR
- run local env
- observe new icon in docs
